pynestml/codegeneration/nest_codegenerator.py

```diff
@@ -54,6 +56,7 @@
 from pynestml.meta_model.ast_neuron import ASTNeuron
 from pynestml.meta_model.ast_node_factory import ASTNodeFactory
```

Comment:
I see in my IDE that some of the imports are unused. Could you remove them as well?